supertokens_python/recipe/emailpassword/utils.py

```diff
@@ -70,11 +70,11 @@ async def default_password_validator(value: str) -> Union[str, None]:
     return None
 
 
-async def default_email_validator(value: str) -> Union[str, None]:
+async def default_email_validator(value: Any) -> Union[str, None]:
     # We check if the email syntax is correct
     # As per https://github.com/supertokens/supertokens-auth-react/issues/5#issuecomment-709512438
     # Regex from https://stackoverflow.com/a/46181/3867175
-    if (
+    if (not isinstance(value, str)) or (
```

Comment:
this is wrong!! if it's not a string, then it will return None -> no error?!

Reply:
I think you misinterpreted it.
If `value` isn't a string, `not isinstance(value, str)` will be `True` this means it will return `"Email is not valid"`